djreact/djangoapps/trips/tests/test_models.py

```diff
@@ -156,3 +159,62 @@ def test_delete_facility(self):
         self.assertEqual(1, len(Facility.objects.all()))
         self.facility.delete()
         self.assertEqual(0, len(Facility.objects.all()))
+
+
+class TestTrip(TestCase):
+    """Test the application Trip model for CURD oprations"""
+
+    def setUp(self):
+        """Setup objects for testing"""
+
+        location_1 = LocationFactory()
+        self.trip = TripFactory.create(locations_included=[location_1], starting_location=location_1)
+
+    def test_create(self):
+        """Checks if setup has created model object"""
+        self.assertIsNotNone(self.trip.id)
+
+    def test_update(self):
+        """Test update trip method."""
+        self.trip.description = 'This is my dummy description'
+        self.trip.save()
+        trip = Trip.objects.get(id=self.trip.id)
+        self.assertEqual(trip.description, self.trip.description)
```

Comment:
we can additionally check if the `updated` field is getting the time of update, just in case.